examples/node/main.rs

```diff
@@ -18,24 +26,33 @@ static BASE_DIR: &str = ".ldk";
 static LOG_FILE: &str = "logs.txt";
 
 fn main() {
-    dotenv::from_path("examples/node/.env").unwrap();
+    init_logger();
+    info!("Logger initialized");
+
+    let lsp_address = "http://127.0.0.1:6666".to_string();
+    info!("Contacting lsp at {} ...", lsp_address);
+    let lsp_auth_token =
+        "iQUvOsdk4ognKshZB/CKN2vScksLhW8i13vTO+8SPvcyWJ+fHi8OLgUEvW1N3k2l".to_string();
+    let lsp_client = LspClient::connect(lsp_address, lsp_auth_token);
+    let lsp_info = lsp_client.channel_information();
+    let lsp_info = ChannelInformationReply::decode(&*lsp_info).unwrap();
+    let ln_node_address = NodeAddress {
+        pub_key: lsp_info.pubkey,
+        address: lsp_info.host,
+    };
+    info!("Lsp pubkey: {}", lsp_info.lsp_pubkey.to_hex());
+    info!("LN node {:?}", ln_node_address);
 
     // Create dir for node data persistence.
     fs::create_dir_all(BASE_DIR).unwrap();
```

Comment:
Need to move this to the beginning of `main()` otherwise the app crashes when `BASE_DIR` doesn't exist